### PR TITLE
Update private network access CORS information

### PR DESCRIPTION
**Pull request by erik-anderson**

### CODE REVIEW

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ This table lists:
...
+| Send CORS preflight requests for private network access | v104 | | Starting with v104, Microsoft Edge sends a CORS [preflight](https://developer.chrome.com/blog/private-network-access-preflight/) request before a page from the internet is allowed to request resources from a local network (intranet).  The intranet server should respond to the preflight by providing explicit permission to access the resource.  The result of this check is not yet enforced.  Enforcement will begin in Edge 111 at the earliest.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see the [Chrome Platform Status entry](https://chromestatus.com/feature/5737414355058688) and [_Chrome Developers_ blog post](https://developer.chrome.com/blog/private-network-access-preflight/#rollout-plan).  Two compatibility policies are available to suppress the CORS preflight request: [InsecurePrivateNetworkRequestAllowed](/deployedge/microsoft-edge-policies#insecureprivatenetworkrequestsallowed) and [InsecurePrivateNetworkRequestAllowedForUrls](/deployedge/microsoft-edge-policies#insecureprivatenetworkrequestsallowedforurls). |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:51]:
```suggestion
| Send CORS preflight requests for private network access | v104 | | Starting with v104, Microsoft Edge sends a CORS [preflight](https://developer.chrome.com/blog/private-network-access-preflight/) request before a page from the internet is allowed to request resources from a local network (intranet).  The intranet server should respond to the preflight by providing explicit permission to access the resource.  The result of this check is not yet enforced.  Enforcement will begin in Microsoft Edge 111 at the earliest.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see the [Chrome Platform Status entry](https://chromestatus.com/feature/5737414355058688) and [_Chrome Developers_ blog post](https://developer.chrome.com/blog/private-network-access-preflight/#rollout-plan).  Two compatibility policies are available to suppress the CORS preflight request: [InsecurePrivateNetworkRequestAllowed](/deployedge/microsoft-edge-policies#insecureprivatenetworkrequestsallowed) and [InsecurePrivateNetworkRequestAllowedForUrls](/deployedge/microsoft-edge-policies#insecureprivatenetworkrequestsallowedforurls). |
```

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 2]
```diff
@@ -42,13 +42,13 @@ This table lists:
...
+| Deprecate WebRTC's Plan B SDP semantics | 104 (Chrome+2) | | This change is happening in the Chromium project, on which Microsoft Edge is based.  This change deprecates a legacy Session Description Protocol (SDP) dialect called Plan B.  This SDP format is being replaced by the Unified Plan, which is a spec-compliant and cross-browser compatible SDP format.  For more information, see the [Chrome Platform Status entry](https://www.chromestatus.com/feature/5823036655665152), [PSA: Plan B should throw in M96 Beta and Stable](https://groups.google.com/g/discuss-webrtc/c/zRIgxG18D80/m/k4ZPzBO3AAAJ), and [PSA: Plan B throwing in Stable and Extended Deprecation Trial End Date](https://groups.google.com/u/1/g/discuss-webrtc/c/gEHrZyYKsfU).  The Microsoft rollout schedule for deprecation is planned for two releases after Chrome.  Requesting a [WebRTC Plan B Reverse Origin Trial Token](https://developer.chrome.com/origintrials/#/view_trial/3892235977954951169) allows sites to continue to use the deprecated API until v101. |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:45]:
```suggestion
| Deprecate WebRTC's Plan B SDP semantics | v104 (Chrome+2) | | This change is happening in the Chromium project, on which Microsoft Edge is based.  This change deprecates a legacy Session Description Protocol (SDP) dialect called Plan B.  This SDP format is being replaced by the Unified Plan, which is a spec-compliant and cross-browser compatible SDP format.  For more information, see the [Chrome Platform Status entry](https://www.chromestatus.com/feature/5823036655665152), [PSA: Plan B should throw in M96 Beta and Stable](https://groups.google.com/g/discuss-webrtc/c/zRIgxG18D80/m/k4ZPzBO3AAAJ), and [PSA: Plan B throwing in Stable and Extended Deprecation Trial End Date](https://groups.google.com/u/1/g/discuss-webrtc/c/gEHrZyYKsfU).  The Microsoft rollout schedule for deprecation is planned for two releases after Chrome.  Requesting a [WebRTC Plan B Reverse Origin Trial Token](https://developer.chrome.com/origintrials/#/view_trial/3892235977954951169) allows sites to continue to use the deprecated API until v101. |
```

Comment (mikehoffms) [reply microsoft-edge/web-platform/site-impacting-changes.md:45]: @erik-anderson - is "v101" at bottom of the line still correct?

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 3]
```diff
@@ -42,13 +42,13 @@ This table lists:
...
+| Deprecate WebRTC's Plan B SDP semantics | v104 (Chrome+2) | | This change is happening in the Chromium project, on which Microsoft Edge is based.  This change deprecates a legacy Session Description Protocol (SDP) dialect called Plan B.  This SDP format is being replaced by the Unified Plan, which is a spec-compliant and cross-browser compatible SDP format.  For more information, see the [Chrome Platform Status entry](https://www.chromestatus.com/feature/5823036655665152), [PSA: Plan B should throw in M96 Beta and Stable](https://groups.google.com/g/discuss-webrtc/c/zRIgxG18D80/m/k4ZPzBO3AAAJ), and [PSA: Plan B throwing in Stable and Extended Deprecation Trial End Date](https://groups.google.com/u/1/g/discuss-webrtc/c/gEHrZyYKsfU).  The Microsoft rollout schedule for deprecation is planned for two releases after Chrome.  Requesting a [WebRTC Plan B Reverse Origin Trial Token](https://developer.chrome.com/origintrials/#/view_trial/3892235977954951169) allows sites to continue to use the deprecated API until v101. |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:45]: @erik-anderson - is "v101" correct at end?

Comment (JasonAndrewWriter) [reply microsoft-edge/web-platform/site-impacting-changes.md]: @mikehoffms reverted

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**